taprpc/mintrpc/mint.swagger.json

```diff
@@ -663,11 +659,14 @@
     "protobufAny": {
       "type": "object",
       "properties": {
-        "@type": {
+        "type_url": {
```

Comment:
This shifted due to a version bump?

Reply:
Should be fixed now in my latest commits. I've checked to make sure that we require the same dep versions in taprpc and the main go.mod